File: FrontEnd/src/models/produto.ts
export class Produto {
    public id: number;
    public nome: string;
    public preco: number;
    public estoque: number;

    constructor(
        nome: string,
        preco: number,
        estoque: number) {
        this.nome = nome;
        this.preco = preco;
        this.estoque = estoque;
    }
}

File: BackEnd/CadastroProduto.Worker/Models/Produto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CadastroProduto.Worker
{
    public class Produto
    {
        public Guid Guid { get; set; }
        public string Nome { get; set; }
        public decimal Preco { get; set; }
        public long Estoque { get; set; }
    }
}
